scene/gui/link_button.cpp

```diff
@@ -157,8 +157,11 @@ void LinkButton::_notification(int p_what) {
 			ERR_FAIL_COND(ae.is_null());
 
 			DisplayServer::get_singleton()->accessibility_update_set_role(ae, DisplayServer::AccessibilityRole::ROLE_LINK);
-			if (!xl_text.is_empty() && get_accessibility_name().is_empty()) {
+			const String &ac_name = get_accessibility_name();
+			if (!xl_text.is_empty() && ac_name.is_empty()) {
 				DisplayServer::get_singleton()->accessibility_update_set_name(ae, xl_text);
+			} else if (xl_text.is_empty() && ac_name.is_empty() && !get_tooltip_text().is_empty()) {
+				DisplayServer::get_singleton()->accessibility_update_set_name(ae, get_tooltip_text()); // Fallback to tooltip.
```

Comment:
Nitpick: "Fallback" is the noun, "fall back" is the action.
I just looked it up and a lot of comments make the same mistake, so this may either be a very stupid nitpick or something I don't understand about development.
```suggestion
				DisplayServer::get_singleton()->accessibility_update_set_name(ae, get_tooltip_text()); // Fall back to tooltip.
```